sampler_t.adoc

```diff
@@ -46,7 +46,7 @@ The sampler fields are described in the table below:
 
 |
 
-Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized or unnormalized values.
+Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized ([0,1[) or unnormalized ({0, size-1}) values.
```

Comment:
Wow, that was fast :-)

This was the result from [my testing](https://www.milania.de/blog/Addressing_mode_of_sampler_objects_for_image_types_in_OpenCL_reviewed). Relevant lines from the AMD output for accessing an 1D image with values 1,...,10:
```
 0.90000 |              10 |     10 |            10 |    10
 1.00000 |              10 |      1 |            10 |     0
```
This led me to the conclusion that 1.0 already gives an out of border value and therefore the right side is not included (hence `[`).

`[]` because by using normalised coordinates all continuous values between 0 and 1 are possible and `{}` because only the discrete values `{0,1,2,...,size-1}` are valid.